etc/kayobe/kolla.yml

```diff
@@ -15,7 +15,7 @@ kolla_source_url: https://github.com/stackhpc/kolla.git
 
 # Version (branch, tag, etc.) of Kolla source code repository if type is
 # 'source'.
-kolla_source_version: refs/tags/stackhpc/7.0.2.6-rc
+kolla_source_version: refs/tags/stackhpc/7.0.2.6
```

Comment:
This doesn't exist right now - are you in the process of creating it?

Reply:
Indeed. It exists now.